role-service/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,22 @@
+spring:
+  application:
+    name: role-service
+  cloud:
+    config:
+      # If cannot connect to config server, don't start the application
+      fail-fast: true
+      discovery:
+        enabled: true
+        service-id: CONFIG-SERVER
```

Comment:
why the service-id is config-server

Reply:
is it the place where we ASK for the config-server?

Reply:
Yes, exactly, and that's the name the config server is being registered with the eureka server